certbot-ci/certbot_integration_tests/utils/acme_server.py

```diff
@@ -52,14 +58,15 @@ def setup_acme_server(acme_server, nodes, proxy=True):
     """
     acme_type = 'pebble' if acme_server == 'pebble' else 'boulder'
     acme_xdist = _construct_acme_xdist(acme_server, nodes)
-    workspace, stop = _construct_workspace(acme_type)
+    workspace, server_cleanup = _construct_workspace(acme_type)
```

Comment:
I think the body of `_construct_workspace` can be simplified and cleaned up a bit with the removal of traefik.

Reply:
Done.